### ISSUE TITLE
Not allowing to work with Node 8

### ISSUE THREAD

**Issue by thecodejack** [EnoahNetzach/extract-scss-variables#5]
I already see that changes are merged for supporting node 8. But these aren't published yet. Can you please publish a version of it?

**Comment by donni106** [comment]
Yes, I also encountered the problem with not supported node version.
@EnoahNetzach can you please publish the new version to npm?
